resources/auth/register.py

```diff
@@ -0,0 +1,28 @@
+import bcrypt
+from flask_restful import Resource, request
+from models.ConnectionPool import ConnectionPool
+from flask_jwt_extended import create_access_token
+from datetime import timedelta
+
+class Register(Resource):
+    def post(self):
+        pool = ConnectionPool()
+
+        email = request.json.get('email')
+        password = request.json.get('password')
+        
+        if not email:
+            return 'Missing field: email', 400
+        if not password:
+            return 'Missing field: password', 400
+        
+        hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+
```

Comment:
Should probably confirm that there is no existing user in the database with provided email.

Reply:
Email field in the DB is set to unique so when a user attempts to register with a duplicate email the error handling in the ConnectionPool should throw and return an error to the developer.